### PR TITLE
React & Redux group project - Space Travelers' Hub

### PR DESCRIPTION
**Pull request by benshidanny11**
### In this project  we : 
:heavy_check_mark: Implemented basic project setup (React, Redux, Folder structure)
:heavy_check_mark: Fetched data from APIs and updated Redux store
:heavy_check_mark: Populated data from redux store to react components
:heavy_check_mark: Wrote actions and reducers for booking rockets and joining missions
:heavy_check_mark: Wrote actions and reducers for canceling rockets/dragons and leaving missions
:heavy_check_mark: Implemented conditional component rendering
:heavy_check_mark: Implemented my profile section (Displayed reserved rockets and missions)
:heavy_check_mark: Wrote actions and reducers for canceling rockets/dragons and leaving missions
:heavy_check_mark: Wrote actions and reducers for canceling rockets/dragons and leaving missions
:heavy_check_mark: Implemented unit tests

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -8,12 +8,15 @@ import MyProfile from './pages/MyProfile';
...
   return (
     <div>
-      <Head />
-      <Routes>
-        <Route exact="true" path="/" element={<Rockets />} />
-        <Route exact="true" path="/missions" element={<Missions />} />
-        <Route exact="true" path="/profile" element={<MyProfile />} />
-      </Routes>
+      <BrowserRouter>
+        <Head />
```
Comment (keneogu) [inline src/components/App.js]: - [ ] <NavLink  /> should be used for navbar...Let's make this look more professional change this to <NavLink /> and go to the pages folder and rename NavBar.js to follow a great naming convention.

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @benshidanny11 & @KanoCode, 

Good job so far!

- All linters are passing
- You are following GitHub flow
- Good commit history
- Good PR description
- Nice design

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- [ ] Please, add how to run the tests on the readme file.

_Check the comments under the review._

## Optional suggestions

- [OPTIONAL] In the Profile Section, add the "Cancel reservation" and "Leave Mission" buttons to the lists here. Clicking them should dispatch the actions you have already used in the main Rockets and Missions sections. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/App.js (hunk 1 above)
Comment (benshidanny11) [reply src/components/App.js]: Hello @keneogu,
Thanks for providing us with feedback, we are going to implement them.

File: src/components/App.js [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
   return (
     <div>
-      <Head />
-      <Routes>
-        <Route exact="true" path="/" element={<Rockets />} />
-        <Route exact="true" path="/missions" element={<Missions />} />
-        <Route exact="true" path="/profile" element={<MyProfile />} />
-      </Routes>
+      <BrowserRouter>
+        <NavBar />
```
Comment (mateo951) [inline src/components/App.js:12]: Addressed 🙌

**Review by mateo951 — APPROVED** [review]
### Hi Team, 

Well done making the changes requested by the previous reviewer 🔥
Your project is complete!  There is nothing else to say other than... it's time to merge it ✔ 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**